nengo_dl/tests/test_converter.py

```diff
@@ -839,7 +839,12 @@ def test_spiking_activation(Simulator, seed, rng):
 
     # note: it will not match for batch_size > 1, because keras-spiking generates
     # a different initial state for each batch element, which is not possible in nengo
-    _test_convert(inp, x, inp_vals=[inp_val], temporal_model=True)
+    model = tf.keras.Model(inp, x)
+    conv = converter.Converter(model, temporal_model=True)
+    assert conv.verify(training=False, inputs=[inp_val])
+    if not pytestconfig.getoption("--graph-mode"):
+        # the keras-spiking gradient implementation doesn't work in graph mode
+        assert conv.verify(training=True, inputs=[inp_val])
```

Comment:
Is this a temporary thing (i.e. we should be able to fix this in KerasSpiking for TF 2.4), or is it permanent, or do we not know yet?

Reply:
Never mind, I read the comment above. I've added an issue to `KerasSpiking` to look into this: nengo/keras-spiking#10.